plugins/inputs/fibaro/fibaro.go

```diff
@@ -0,0 +1,183 @@
+package fibaro
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/httpjson"
+)
+
+const sampleConfig = `
+  ## Required Fibaro controller address/hostname.
+  ## Note: connection is done over http/80 as Fibaro did not implement https.
+  server = "<controller>"
+  ## Required credentials to access the API (http://<controller/api/<component>)
+  username = "<username>"
+  password = "<password>"
+`
+
+const description = "Read devices value(s) from a Fibaro controller"
+
+// Fibaro contains connection information
+type Fibaro struct {
+	Server   string
+	Username string
+	Password string
+
+	client httpjson.HTTPClient
```

Comment:
Lets remove this and just use `*http.Client` directly.  We are deprecating the httpjson input in 1.6, so we won't want to depend on it in a new plugin.  Additionally you are not using it for its original purpose of testing, and I like the way you did your tests more.

Reply:
No sure why I used httpjson as client. Will fix that and the other comments as well asap.